### PR TITLE
Update master to beta5

### PR DESCRIPTION
**Pull request by jeremymeng**
Update master to use final beta5 bits.  Tested on Windows and Linux.

### CODE REVIEW

**Comment by jeremymeng** [comment]
@EMaher In the diffs I don't see anything related to Recommendation service.  Is it off in master branch as well?

File: src/PartsUnlimited.Models/Migrations/20150505210609_InitialMigration.cs [hunk 1]
```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
 using System.Collections.Generic;
```
Comment (twsouthwick) [inline src/PartsUnlimited.Models/Migrations/20150505210609_InitialMigration.cs]: Can you add the license header back?

**Comment by twsouthwick** [comment]
Other than the license headers in the migration files, LGTM.

**Comment by EMaher** [comment]
@jeremymeng Looks like the recommendation service set to true (in config.json) in the the dev/beta5 branch as well as the master.  Users just need to add a key to the service and they can start using it.  No changes need to be made for this.

I agree with @twsouthwick.  Everything looks good except the one missing license header.  Just add that back and you can merge in the changes.

**Comment by jeremymeng** [comment]
This resolves #22.

File: src/PartsUnlimited.WebJobs.ProcessOrder/Functions.cs [hunk 2]
```diff
@@ -20,7 +20,8 @@ public static void CreateOrderProcessTask([Queue("orders")] CloudQueue orderQueu
             Console.WriteLine("Starting Create Order Process Task");
             try
             {
-                var config = new Configuration().AddJsonFile("config.json");
+                var builder = new ConfigurationBuilder(Environment.CurrentDirectory).AddJsonFile("config.json");
```
Comment (jeremymeng) [inline src/PartsUnlimited.WebJobs.ProcessOrder/Functions.cs]: I could use DI in `Program.cs` to get the `ApplicationBasePath`.  But for this static class I am not sure how to achieve that.  Would `Environment.CurrentDirectory` work? @jamescarpinter

Comment (jeremymeng) [inline src/PartsUnlimited.WebJobs.ProcessOrder/Functions.cs]: I found another way to do it, which should achieve the same goal as the original code.

**Comment by EMaher** [comment]
Looks good.

**Pull request merged**